time t
string arm
uint32 palstance
uint32 line_speed
string meter
uint32 brush
uint32 big_brush
bool lock
